lib/world_cup.rb

```diff
@@ -0,0 +1,50 @@
+require './lib/player'
+require './lib/team'
+require 'pry'
+
+class WorldCup
+  attr_reader :year, :teams
+
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+    @player_hash = {}
+  end
+
+  def active_players_by_position(position)
+    french_players = @teams[0].players
+    croatian_players = @teams[1].players
+    active_players = []
+
+    while @teams[0].eliminated? == false
+      french_players.find_all do |player|
```

Comment:
You already wrote a method in your team class to find players by position. You should use it